app/controllers/support/locations_controller.rb

```diff
@@ -8,6 +8,21 @@ def index
       redirect_to support_providers_path
     end
 
+    def new
+      provider
```

Comment:
do we need to set the provider first? 
perhaps worth removing it from `edit` as well if we don't, or into a before action for `new` and `edit`

Reply:
Nope, forgot to remove that. FYI We've opted to use memoised/lazily invoked methods for these sort of situations  - there's an ADR for it here https://github.com/DFE-Digital/teacher-training-api/blob/master/docs/adr/0004-before-action-variables.md.